### PR TITLE
Add gs-tama

### PR DESCRIPTION
**Pull request by sguizard**
Add gs-tama (Gene-Switch Transcriptome Annotation by Modular Algorithms) package
Forked from https://github.com/GenomeRIK/tama

### CODE REVIEW

**Comment by sguizard** [comment]
@BiocondaBot please add label

File: recipes/gs-tama/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  sha256: 11e7e1aadc5a9a5f8eb4b45aedc8217d937e15fa59b2ad7f1904c32a08c76062
+
+requirements:
+  run:
+    - python=2.7.18
+    - biopython=1.76
+    - pysam=0.16.0.1
+    - bedtools=2.30.0
+    - blast=2.11.0
+    - samtools=1.12
```
Comment (dpryan79) [inline recipes/gs-tama/meta.yaml:22]: Are these exact versions required, or might newer versions work as well?

Comment (sguizard) [reply recipes/gs-tama/meta.yaml:22]: It should work fine with newer versions. 
I just set up those because it the versions with I tested the package.

Comment (dpryan79) [reply recipes/gs-tama/meta.yaml]: Am I correct that this is compatible with python3?

Comment (sguizard) [reply recipes/gs-tama/meta.yaml]: It has been developed for python2. 
I didn't tested the compatibility with python3. 
Maybe I'll able to check that this coming weeks.

**Review by dpryan79 — APPROVED** [review]

**Comment by sguizard** [comment]
Thanks for your help and your approval

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**